### PR TITLE
new hero  - update layout to match responsive mock up

### PR DESCRIPTION
**Pull request by moog16**
also moved localConfig from HeroComponent to ComponentCatalogPanel. This allowed me to decouple the HeroOptions component to match the hero.

Still outstanding:
- Fix 100% height of the hero.
- test old hero components

### CODE REVIEW

File: src/ComponentCatalogPanel.js [hunk 1]
```diff
@@ -2,17 +2,60 @@ import React, {Component} from 'react';
...
 // ComponentCatalogPanel is the container for catalog component content,
 // that renders the hero and demo sections.
 class ComponentCatalogPanel extends Component {
   ripples = [];
   initRipple = el => el && this.ripples.push(new MDCRipple(el));
 
+  constructor(props) {
+    super(props);
+    // Deep copy for local object
+    const localConfig = JSON.parse(JSON.stringify(this.props.initialConfig));
```
Comment (williamernest) [inline src/ComponentCatalogPanel.js]: This causes issues with component Hero sections that don't have options.

Comment (moog16) [reply src/ComponentCatalogPanel.js]: ah...ok I'll fix that. Do the rest of the changes look ok to you?

Comment (williamernest) [reply src/ComponentCatalogPanel.js]: Yeah, I think it looks great. I pushed some styles for max width so the card web tab doesn't expand to the entire screen but I definitely think it looks better.

**Comment by moog16** [comment]
merging. disregard cla since its just me

**Pull request merged**